modules/contact/test/tests/simple_contact/two_block_compress/tests

```diff
@@ -11,6 +11,17 @@
     min_ad_size = 100
     abs_zero = 1.0e-8
   []
+  [frictionless-compress-2d_pg]
+    type = 'CSVDiff'
+    input = 'two_equal_blocks_compress_2d_pg.i'
+    csvdiff = 'two_equal_blocks_compress_2d_pg_out.csv'
+    mesh_mode = 'REPLICATED'
+    requirement = 'The system shall simulate correct contact behavior in 2D when two blocks with the '
+                  'same height come into contact using the Petrov-Galerkin approach'
```

Comment:
```suggestion
                  'same height come into contact using the Petrov-Galerkin approach.'
```
Requirements should always be complete sentences

Reply:
Got it! Let me fix those and some others that seem to missing it as well